evalml/tests/component_tests/test_column_selector_transformers.py

```diff
@@ -31,7 +47,7 @@ def test_column_transformer_empty_X(class_to_test):
     assert_frame_equal(X, transformer.fit_transform(X))
 
     transformer = class_to_test(columns=["not in data"])
-    with pytest.raises(ValueError, match="'not in data' not found in input data"):
+    with pytest.raises(ValueError, match="not found in input data"):
```

Comment:
This test doesn't actually test that a column type does not exist, but rather that a column does not exist! Since we're passing in the keyword `columns`, we trigger `self._check_input_for_columns(X)` to raise a ValueError, but don't we actually want to test the following:
```
transformer = SelectByType(column_types=["some fake column type"])
```
and see that the ValueError is raised? I think this is related to my previous comment about using `_check_input_for_columns` :)